src/content/en/fundamentals/engage-and-retain/push-notifications/notification-behaviour.md

```diff
@@ -0,0 +1,255 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-03-03 #}
+{# wf_published_on: 2016-06-30 #}
+
+# Notification Behaviour {: .page-title }
+
+{% include "web/_shared/contributors/mattgaunt.html" %}
+
+
+
+So far we've looked at the options that alter the visual appearance of a notification. There
+are options that alter the behaviour of notifications.
+
+Be default, calling `showNotification()` with just visual options will have
+the following behaviours:
+
+1. Clicking on the notification does nothing.
+1. Each new notification is shown one after the other. The browser will not  collapse the
+notifications in anyway.
+1. The platform may play a sound or vibration the users devices (depending on the platform).
+1. On some platforms the notification will disappear after a short
+period of time while others will show the notification unless the user interacts with it (i.e.
+compare Android and Desktop with your notifications).
+
+In this section we are going to look at how we can alter these default behaviours using options
+alone. These are relatively easy to implement and take advantage of.
+
+### Notification Click Event
+
+When a user clicks on a notification the default behaviour is for nothing
+to happen, it doesn't even close / remove the notification.
+
+The common practice for a notification click is for it to close and perform some other logic
+(i.e. open a window or make some API call to the application).
+
+To achieve this we need to add a 'notificationclick' event listener to our service worker. This
+will be called when ever a notification is clicked.
+
+    self.addEventListener('notificationclick', function(event) {
+      const clickedNotification = event.notification;
+      clickedNotification.close();
+
+      // Do something as the result of the notification click
+      const promiseChain = doSomething();
+      event.waitUntil(promiseChain);
+    });
+
+As you can see in this example, the notification was clicked can be accessed via the
+`event.notification parameter`. From this we can via the properties on the notification, in
```

Comment:
'From this we can via the properties...' Huh?
'In this case' is the start of a new sentence.